tests/foreman/api/test_contentviewfilter.py

```diff
@@ -65,43 +62,6 @@ def content_view_module_stream(module_org, sync_repo_module_stream, module_targe
 class TestContentViewFilter:
     """Tests for content view filters."""
 
-    @pytest.mark.tier2
-    def test_negative_get_with_no_args(self, target_sat):
-        """Issue an HTTP GET to the base content view filters path.
-
-        :id: da29fd90-cd96-49f9-b94e-71d4e3a35a57
-
-        :expectedresults: An HTTP 200 response is received if a GET request is
-            issued with no arguments specified.
-
-        :CaseImportance: Low
-        """
-        response = client.get(
-            target_sat.api.AbstractContentViewFilter().path(),
-            auth=get_credentials(),
-            verify=False,
-        )
-        assert response.status_code == http.client.OK
-
-    @pytest.mark.tier2
-    def test_negative_get_with_bad_args(self, target_sat):
```

Comment:
Is this scenario already been covered in any other test? Any reason to remove?

Reply:
I believe this was removed as it was considered a low value test case